src/transformState.js

```diff
@@ -5,7 +5,27 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const order of actions) {
+    switch (true) {
+      case order.type === 'addProperties':
+        Object.assign(state, order.extraData);
+        break;
+
+      case order.type === 'removeProperties':
+        for (const toBeRemoved of order.keysToRemove) {
+          if (toBeRemoved in state) {
+            delete state[toBeRemoved];
+          }
+        }
+        break;
+
+      default:
```

Comment:
So if we receive an action with a type for example `addProperty` we will delete everything?!

Reply:
Excelent advice ! Have implemented addtional  swithc case to handle 'clear' action.